playbooks/update-dns-records.yml

```diff
@@ -3,7 +3,6 @@
 - name: 'Update DNS records'
   hosts: localhost
```

Comment:
Should this be localhost or should it point to wherever we are hosting our dns server?

Reply:
It should be the system used to manage the DNS records, and this is typically **not** the dns server. Often it is `localhost`, but it could be elsewhere. You have a good point, however, as in general, I've started to move away from "hardcoded" `localhost` in these playbooks, and rather just use the inventory to point to the `localhost`, so I'll get this updated to something like `dns-management-host` or similar that we can drive through the inventory.

Reply:
Changes made: https://github.com/redhat-cop/infra-ansible/pull/259/commits/6a24e6c5fd130570e3ed4bf35ec8a255cb3cf4ea